harness/pyproject.toml

```diff
@@ -1,4 +1,86 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = ["setuptools>=64", "wheel", "setuptools_scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "determined"
+description = "Determined AI: The fastest and easiest way to build deep learning models."
+requires-python = ">=3.8"
+
+authors = [
+  { name = "Determined AI", email = "ai-open-source@hpe.com" },
+]
+
+dependencies = [
```

Comment:
Does pip support reading dependencies from here yet, or is including requirements as a dynamic section from file finally out of beta?  I could read the docs, but I'd rather make a comment on a PR. :D

Reply:
I.e. in lieu of of `requirements.txt`?

Reply:
Yup

Reply:
It does not, as far as I understand.